Build.mak

```diff
@@ -1,6 +1,3 @@
-# This will make D2 unittests fail if stomping prevention is triggered
-export ASSERT_ON_STOMPING_PREVENTION=1
-
```

Comment:
Guess what - `-cov` implementation in runtime triggers the assertion and there is no way to change it apart from tweaking runtime :)

Though I had to disable D2 `-cov` bit because of another bug anyway, will restore that now.